mmdet/integration/nncf/compression.py

```diff
@@ -173,7 +173,8 @@ def _get_fake_data_for_forward(cfg, nncf_config, get_fake_input_func):
         assert len(input_size) == 4 and input_size[0] == 1
         H, W, C = input_size[2], input_size[3], input_size[1]
         device = next(model.parameters()).device
-        return get_fake_input_func(cfg, orig_img_shape=tuple([H, W, C]), device=device)
+        with no_nncf_trace():
+            return get_fake_input_func(cfg, orig_img_shape=tuple([H, W, C]), device=device)
```

Comment:
2 @aleksei-kashapov : Alexei, are you sure that `no_nncf_trace` is required here?

Reply:
so the reason why we should add here `no_nncf_trace()` is following:
NNCF traces all pytorch operations if there is `no_nncf_trace()` context. Therefore, the operations that are inside `get_fake_input_func() (torch.stack)` are gotten to a model's graph which is not correct. 

Working with GPU it wasn't a problem because these operations don't have any connection with other nodes. However, in CPU mode it becomes the first node in the model's graph. This leads to a statistics collection error

Reply:
Ok, I see